### ISSUE TITLE
Self-service checklist flow for techs & instructors

### ISSUE THREAD

**Issue by smartin015** [protohaven/protohaven_api#49]
Onboarding new techs and instructors is a huge source of operational load. Once someone is approved, it should require very minimal effort on the part of the leads and staff to get them through the paperwork and up to speed. 

I'm imagining a new Airtable that stores a list of questions, plus a "submissions" table that tracks shop techs' progress on the question list. We can periodically invalidate the submissions table and prompt techs to re-orient themselves to ensure knowledge transfer.

**Comment by smartin015** [comment]
Closing in favor of #55 to detect validation issues and prompt the tech to fix them
